actioncable/lib/action_cable/engine.rb

```diff
@@ -53,8 +53,12 @@ class Engine < Rails::Engine # :nodoc:
 
     initializer "action_cable.set_work_hooks" do |app|
       ActiveSupport.on_load(:action_cable) do
+        work_executor = (!app.config.cache_classes && app.config.reload_classes_only_on_change) ?
+                        app.reloader :
+                        app.executor
```

Comment:
Do we need both `app.config.cache_classes && app.config.reload_classes_only_on_change`? Can you also oneline app.reloader and app.executor? I had a hard time reading it the way it is now.

Reply:
I think we can always use the reloader. If cache_classes is true the reloader works in the same way of the executor.